types/node/v12/async_hooks.d.ts

```diff
@@ -20,7 +34,7 @@ declare module "async_hooks" {
          * @param triggerAsyncId the unique ID of the async resource in whose execution context this async resource was created
          * @param resource reference to the resource representing the async operation, needs to be released during destroy
          */
-        init?(asyncId: number, type: string, triggerAsyncId: number, resource: Object): void;
+        init?(asyncId: number, type: string, triggerAsyncId: number, resource: object): void;
```

Comment:
changed to match:
https://github.com/peterblazejewicz/DefinitelyTyped/blob/feat/45103/types/node/async_hooks.d.ts#L37